src/js/components/Carousel/doc.js

```diff
@@ -50,6 +55,11 @@ export const themeDoc = {
     type: 'string',
     defaultValue: undefined,
   },
+  'carousel.icons.disabledColor': {
```

Comment:
I think it would be slightly more consistent with the rest of the theme structure to put this at `carousel.disabled.icons.color`.